SharedLib/NpcFinder/NpcNameFinder.cs

```diff
@@ -42,6 +42,9 @@ public class NpcNameFinder
         public int TargetCount { private set; get; }
         public bool MobsVisible => npcs.Count > 0;
         public bool PotentialAddsExist { get; private set; }
+
```

Comment:
With this construct, for it is possible to toggle the state during runtime. Since the `Enabled` property is used in the `NpcNameFinder.Update()` loop.

Is that what you intended to do in the future ?

Personally i would rather create an `INpcNameFinder` interface and implement the public functions what already used in the project. 

Check out how the [Blacklist](https://github.com/Xian55/WowClassicGrindBot/tree/main/Core/Blacklist) is implemented in the project. That way to avoid `Enabled` property and creating a missleading `Enable()` and `Disable()` functions

Reply:
Not actually. I simply mistook it that the NpcNameFinder is fixed in every run.